src/main/resources/application.properties

```diff
@@ -3,14 +3,14 @@ management.endpoints.web.exposure.include=health,metrics,prometheus,liveness
 opentracing.jaeger.log-spans = false
 
 # Server configuration
-server.port = ${SERVICE_PORT:8080}
+server.port = ${SERVICE_PORT:8081}
 
 management.port = ${MANAGEMENT_PORT:8090}
 
 spring.application.name = auth
 
 security.ignored = /*.css
 
-jwt.sharedSecret = ${HS256_KEY:}
```

Comment:
Keep the env variable rather than passing the value directly

Reply:
reverted the file